languages/ruby/outline.scm

```diff
@@ -91,16 +91,16 @@
         (call
             method: (identifier) @name
             arguments: (argument_list . [
-                    (string) @name
-                    (simple_symbol) @name
-                    (scope_resolution) @name
-                    (constant) @name
+                    (string) @context.extra
+                    (simple_symbol) @context.extra
+                    (scope_resolution) @context.extra
+                    (constant) @context.extra
                     "," @context
```

Comment:
I wonder if this should be `@context.extra` as well?

Reply:
We can't use `context.extra` here and everywhere else because we will lose ability to search through symbols:


https://github.com/user-attachments/assets/0ca77bc0-60e2-4179-8f0d-30a7eee253b2